src/containers/Tenant/ObjectSummary/i18n/en.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Not sure, but seems in this project dot notation is commonly used for i18n namings, not sneak_case

Reply:
In my dreams we will slowly move towards https://nda.ya.ru/t/ZL7Txlvx78RyjV...